### ISSUE TITLE
Consider candidates revealed after one scroll step?

### ISSUE THREAD

**Issue by frivoal** [w3c/csswg-drafts#3366]
---

Migrated from https://github.com/WICG/spatial-navigation/issues/56
Originally created by @hugoholgersson on *Fri, 27 Apr 2018 14:26:33 GMT*

---
When finding the candidate set, Chrome's spatnav includes focusables that would become visible after one (document*) scroll step in the navigated direction.

_\*  We don't do this for scrollable divs but I consider that a bug. I believe spatnav should treat both document scrolling and element scrolling in the same way._

This behavior is not spec'ed. Should we? 

**Downsides**:
- Trickier implementation.

**Upsides**: 
- Better UX.
 1) One keystroke instead of two: [scroll + focus] instead of [scroll] + [focus].
 2) Focused element (and its "focus ring", if enabled) is always visible (it cannot be scrolled out).

**Comment by hugoholgersson** [comment]
In,  #3383 comments are out of order.

**Comment by frivoal** [comment]
@hugoholgersson yes, that's due to a bug in the issue migration script. That's a bit annoying, but is too late to fix now. Referring to the original issue can make it an easier read.
